codes/classical/bits/IPP.yml

```diff
@@ -0,0 +1,30 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+
+
+code_id: ipp
+physical: bits
+logical: bits
+
+
+name: 'Code with Identifiable Parent Property'
+introduced: '\cite{//doi.org/10.1137/S0895480100376848}'
+
+
+description: 'If \(C\) is a \(q\)-ary code of length \(n\) and \(a\) and \(b\) are two codewords, then \(c\) is called a descendant of \(a\) and \(b\) if \(c_i \in \{a_i,b_i\}\), for \(i \in \{1,2,...,n\}\).Code \(C\) with the property that given any descendant \(c\) one can always identify at least one of the parent codewords in \(C\)'
```

Comment:
Don't understand last sentence. You mean for each string c, there exists a pair a,b such that c is their descendant and a is a codeword?

Reply:
Denote by X(C) the set of descendants of all pairs of the codewords. There is a mapping from the set X(C) such that for every z in X(C), its image under this mapping is one of the two parents of z in C.

Importantly, IPP codes are defined for any t ≥ 2 parents, not just for 2 parents, see
https://doi.org/10.1137/S0895480100376848
To modify the definition, please replace the parents a and b with a set of t parents c_1,...,c_t. The mapping defined above still should identify one of the parents of z.